packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -3,16 +3,21 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Button from 'terra-button';
 import ButtonGroup from 'terra-button-group';
+import { injectIntl, intlShape } from 'react-intl';
 import ActionHeaderContainer from './_ActionHeaderContainer';
 import styles from './ActionHeader.module.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
-   * Text to be displayed as the title in the header bar.
+   * Displays a single terra `Collapsible Menu View` child element on the right end of the header.
    */
-  title: PropTypes.string,
+  children: PropTypes.element,
+  /**
+  * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the `@private` syntax so this doesn't show up in the generated prop-types documentation.

```suggestion
  * @private
  * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```